src/ontology/mondo-edit.obo

```diff
@@ -292394,10 +292396,12 @@ xref: UMLS:C1861065 {source="OMIM:613765", source="MONDO:equivalentTo", source="
 is_a: MONDO:0005045 {source="DC-OMIM:613765", source="DOID:0110315", source="MESH:C566044/inferred", source="MONDO:Redundant", source="OMIM:613765"} ! hypertrophic cardiomyopathy
 is_a: MONDO:0016191 ! qualitative or quantitative defects of titin
 is_a: MONDO:0024573 {source="MESH:C566044", source="MONDO:OMIM", source="OMIM:613765"} ! familial hypertrophic cardiomyopathy
+is_a: MONDO:0100494 {source="https://orcid.org/0000-0001-9310-0163"} ! autosomal dominant titinopathy
```

Comment:
Why was this parent term selected vs. `MONDO:0100175` TTN-related myopathy?

Reply:
my bad, I will fix

Reply:
Oh, so 'AD' there was autosomal dominant.... I did not get that until now. Thank you!!!